scratch/PCA/extract_data.ipynb

```diff
@@ -18,40 +18,81 @@
     "\"\"\"\n",
     "\n",
     "from __future__ import division\n",
```

Comment:
Sorry, Henry, but I'm already gone. We can Skype at 6:30 if you like. I'll be in Chicago for the remainder of the week.

Branden

Sent from my iPhone

> On Aug 16, 2016, at 5:06 PM, HGeerlings <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.ipynb:
> 
> > @@ -18,40 +18,81 @@
> >      "\"\"\"\n",
> >      "\n",
> >      "from __future__ import division\n",
> Branden, can you meet briefly? -- Henry
> …
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>